examples/ownable-two-step/tests/ownable_two_step.rs

```diff
@@ -151,6 +153,113 @@ async fn accept_ownership(alice: Account, bob: Account) -> Result<()> {
     Ok(())
 }
 
+#[e2e::test]
+async fn cancel_transfer_ownership(alice: Account, bob: Account) -> Result<()> {
+    let alice_addr = alice.address();
+    let bob_addr = bob.address();
+
+    let contract_addr = alice
+        .as_deployer()
+        .with_constructor(ctr(alice_addr))
+        .deploy()
+        .await?
+        .address()?;
+
+    let contract = Ownable2Step::new(contract_addr, &alice.wallet);
+
+    let receipt = receipt!(contract.transferOwnership(bob_addr))?;
+    assert!(receipt.emits(OwnershipTransferred {
+        previousOwner: alice_addr,
+        newOwner: bob_addr,
+    }));
+
+    let receipt = receipt!(contract.transferOwnership(Address::ZERO))?;
+    assert!(receipt.emits(OwnershipTransferred {
+        previousOwner: alice_addr,
+        newOwner: Address::ZERO,
+    }));
+
+    // Connect as Bob and try to accept ownership
+    let contract = Ownable2Step::new(contract_addr, &bob.wallet);
+    let err = send!(contract.acceptOwnership())
+        .expect_err("should not accept when not pending owner");
+
+    err.reverted_with(Ownable2Step::OwnableUnauthorizedAccount {
+        account: bob_addr,
+    });
+
+    let Ownable2Step::ownerReturn { owner } = contract.owner().call().await?;
+    assert_eq!(owner, alice_addr);
+
+    let Ownable2Step::pendingOwnerReturn { pendingOwner } =
+        contract.pendingOwner().call().await?;
+    assert_eq!(pendingOwner, Address::ZERO);
+
+    Ok(())
```

Comment:
I think this test is more concise and gets to the point. WDYT?

```suggestion
    let alice_addr = alice.address();
    let bob_addr = bob.address();

    let contract_addr = alice
        .as_deployer()
        .with_constructor(ctr(alice_addr))
        .deploy()
        .await?
        .address()?;
    let contract = Ownable2Step::new(contract_addr, &alice.wallet);
    receipt!(contract.transferOwnership(bob_addr))?;

    let receipt = receipt!(contract.transferOwnership(Address::ZERO))?;
    assert!(receipt.emits(OwnershipTransferStarted {
        previousOwner: alice_addr,
        newOwner: Address::ZERO,
    }));

    let Ownable2Step::pendingOwnerReturn { pendingOwner } =
        contract.pendingOwner().call().await?;
    assert_eq!(pendingOwner, Address::ZERO);

    Ok(())
```